src/Transformer.php

```diff
@@ -11,14 +11,20 @@
 
     private $key;
     private $dataToTransform;
+    protected $options;
```

Comment:
Why protected and not private ?

Reply:
Because I want the property to be accessible in transformers to update it during transformations.